README.md

```diff
@@ -203,13 +203,24 @@ passed a ``Featurizer`` object. ``deepchem`` provides a number of
 different subclasses of ``Featurizer`` for convenience:
 
 ### Performances
```

Comment:
Wow, the new numbers look really nice for the graph-conv! Could you update the table with the loading/running times for  the logistic regression and graph-conv models as well?